akka-docs/src/main/paradox/typed/persistence.md

```diff
@@ -646,18 +646,18 @@ reference documentation of the chosen plugin.
 
 ## Example project
 
-@java[@extref[Persistence example project](samples:akka-sample-persistence-java)]
-@scala[@extref[Persistence example project](samples:akka-sample-persistence-scala)]
+@java[@extref[Persistence example project](samples:akka-samples-persistence-java)]
```

Comment:
Wait, so in https://github.com/lightbend/example-code-service/blob/master/example-code-service/conf/akka-templates.conf it is `akka-sample-persistence-java` and the directory in https://github.com/akka/akka-samples/tree/2.6/akka-sample-persistence-java is `akka-sample-persistence-java` - but indeed https://developer.lightbend.com/start/?group=akka&project=akka-samples-persistence-java actually works

where does `akka-samples-persistence-java` come from?

Reply:
I don't know how it works. Maybe example-code-service is concatenating the names? 

https://github.com/lightbend/example-code-service/blob/master/example-code-service/conf/akka-templates.conf#L3

https://github.com/lightbend/example-code-service/blob/master/example-code-service/conf/akka-templates.conf#L98